### PR TITLE
games-util/lutris: Make lutris great again

### PR DESCRIPTION
**Pull request by wisniew**
Continuation of lutris reworking..

New changes:
- Picked all changes from previous RPs.
- Commits are signed off
- Created account on bugzilla

### CODE REVIEW

File: games-util/lutris/lutris-0.5.0.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (Zlogene) [inline games-util/lutris/lutris-0.5.0.1.ebuild]: why not 7?

File: games-util/lutris/lutris-0.5.0.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+PYTHON_COMPAT=( python3_{4,5,6} )
+PYTHON_REQ_USE="sqlite"
+
+inherit distutils-r1 gnome2-utils xdg-utils
+
+DESCRIPTION="Lutris is an open source gaming platform for GNU/Linux."
```
Comment (Zlogene) [inline games-util/lutris/lutris-0.5.0.1.ebuild]: `DESCRIPTION` should not contain a package name and a dot in the end

File: games-util/lutris/lutris-0.5.0.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	SRC_URI="https://github.com/lutris/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3"
+SLOT="0"
+
+RDEPEND="
+	dev-python/dbus-python[${PYTHON_USEDEP}]
+	dev-python/pillow
```
Comment (Zlogene) [inline games-util/lutris/lutris-0.5.0.1.ebuild]: why did not you enforce `${[PYTHON_USEDEP]}` here?

File: games-util/lutris/lutris-0.5.0.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	dev-python/python-evdev[${PYTHON_USEDEP}]
+	dev-python/pyyaml[${PYTHON_USEDEP}]
+	media-sound/fluid-soundfont
+	gnome-base/gnome-desktop[introspection]
+	net-libs/libsoup
+	net-libs/webkit-gtk:4[introspection]
+	sys-process/psmisc
```
Comment (Zlogene) [inline games-util/lutris/lutris-0.5.0.1.ebuild]: you do not need this dependency explicitly

File: games-util/lutris/lutris-0.5.0.1.ebuild [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+	x11-apps/xgamma
+	x11-apps/xrandr
+	x11-libs/gtk+[introspection]"
+
+	python_install_all() {
```
Comment (Zlogene) [inline games-util/lutris/lutris-0.5.0.1.ebuild]: fix formating starting from here and till the end

File: games-util/lutris/lutris-0.5.0.1.ebuild (hunk 1 above)
Comment (wisniew) [reply games-util/lutris/lutris-0.5.0.1.ebuild]: With 7 show me that EAPI=7 is not supported on my system

Comment (Zlogene) [reply games-util/lutris/lutris-0.5.0.1.ebuild]: then your system is totally obsolete and this needs major updates ;)

File: games-util/lutris/lutris-0.5.0.1.ebuild [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+  distutils-r1_python_install_all
+}
+
+pkg_preinst() {
+	gnome2_icon_savelist
+	gnome2_schemas_savelist
+}
+
+python_test() {
+  virtualx nosetests -v
```
Comment (hydrapolic) [inline games-util/lutris/lutris-0.5.0.1.ebuild]: Seems like it's missing the tab

Comment (wisniew) [reply games-util/lutris/lutris-0.5.0.1.ebuild]: tab? where?

Comment (a17r) [reply games-util/lutris/lutris-0.5.0.1.ebuild]: indendation.

File: games-util/lutris/lutris-0.5.0.1.ebuild (hunk 1 above)
Comment (wisniew) [reply games-util/lutris/lutris-0.5.0.1.ebuild]: funtoo user ;)

**Comment by chritting** [comment]
> ValueError: Namespace GnomeDesktop not available
Consider adding gnome-base/gnome-desktop to RDEPEND. Thank you.

**Comment by a17r** [comment]
`ebuild` does not do any dependency resolution, `emerge` does.

**Comment by a17r** [comment]
Thanks for your work, however https://github.com/gentoo/gentoo/pull/10749 is very close to being merged and if anything is left to fix, you should join forces there.

**Pull request closed without merging**